trip_app/conftest.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from django.contrib.auth import get_user_model
+
+from accounts.tests.user_factory import faker
+
+User = get_user_model()
+
+
+@pytest.fixture()
+def signup_data():
+    """ Create signup data for a new user. """
+    data = {
+        "username": faker.user_name(),
+        "password": faker.password(),
+        "first_name": faker.first_name(),
+        "last_name": faker.last_name(),
+        "email": faker.email(),
+        "is_staff": False,
+        "is_active": True,
+    }
+    return data
+
+
+@pytest.fixture
+def create_user(signup_data, client):
+    """ Create a new user.
+        :return: user id, user name and user password.
+        """
+    password = signup_data.get("password")
+    user = User.objects.create_user(**signup_data)
+    return user.id, user.username, password
```

Comment:
Why you return password not from `user.password` - according to your code, there is no any hashing of password when setting password to user, so why not to do this?

Reply:
Actually `_create_user` which is being called by `create_user` of `django/contrib/auth/models.UserManager` class hashes the password and `user.password` will return hashed value, and I need raw value to get the token.